modules/configuring-a-pipeline-server-with-an-external-amazon-rds-db.adoc

```diff
@@ -0,0 +1,57 @@
+:_module-type: PROCEDURE
+
+[id='configuring-a-pipeline-server-with-an-external-amazon-rds-db_{context}']
+= Configuring a pipeline server with an external Amazon RDS database
+
+To configure a pipeline server with an external Amazon Relational Database Service (RDS) database, you must configure {productname-short} to trust the certificates issued by its certificate authorities (CA).
+
+[IMPORTANT]
+====
+If you are configuring a pipeline server for production pipeline workloads, {org-name} recommends that you use an external MySQL or MariaDB database. 
+====
+
+.Prerequisites
```

Comment:
Just to clarify, does the external Amazon RDS database also need to exist, and therefore would need to be listed as a pre-req, or is it created automatically?

Reply:
Discussed offline - leaving these as-is for now, will update in Q1 after further discussion/confirmation.